app/javascript/src/components/Dashboard/Notes/Data/notes.js

```diff
@@ -0,0 +1,17 @@
+export const notes = [
+  {
+    title: "How to claim warranty",
+    body: `"Are you getting my text???" She texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    status: "Created"
+  },
+  {
+    title: "How to claim warranty",
+    body: `"Are you getting my text???" She texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    status: "Drafted"
+  },
+  {
+    title: "How to claim warranty",
+    body: `"Are you getting my text???" She texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    status: "Drafted"
+  }
+];
```

Comment:
When you are exporting constants from a `constants` file, please keep in mind the following suggestions - 
1. Name the file as `constants.js`. 
2. Keep it in the scope of a directory if the constant is specific to a feature. If it's shared between multiple directories then create a `constants.js` file and define an alias for it. 
3. Whenever you are exporting constants from a file, use the naming convention like this - `export const NOTES_DATA` .